models/src/main/java/com/vimeo/networking2/Pageable.kt

```diff
@@ -0,0 +1,33 @@
+package com.vimeo.networking2
+
+/**
+ * All lists that can be paged should implement this interface to provide
+ * paging urls, total number of items returns for each page and the data for each page.
+ */
+interface Pageable<DATA_T> {
+
+    /**
+     * Total number of items returned.
```

Comment:
Since we have documentation on this interface, I wonder if it is necessary to have documentation on each of the subclasses.